src/interfacers/EmonHubKNXInterfacer.py

```diff
@@ -0,0 +1,273 @@
+import time
+import json
+import re
+import Cargo
+import serial
+import struct
+import asyncio
+import concurrent.futures
+import threading
+
+from emonhub_interfacer import EmonHubInterfacer
+from xknx import XKNX
```

Comment:
xknx should be integrated in the dependencies....or maybe just a mention in the interfacer README